client/lib/eventHubSender.ts

```diff
@@ -248,6 +248,10 @@ export class EventHubSender extends LinkEntity {
         throw new Error("data is required and it must be of type object.");
       }
 
+      if (data.partitionKey){
+        data.partitionKey = (typeof data.partitionKey !== "string") ? (+data.partitionKey).toString() : data.partitionKey;
```

Comment:
In this case using `+` doesn't make sense.

Reply:
Ohh yes,  forgot to remove this.